homeassistant/components/rfxtrx/__init__.py

```diff
@@ -115,8 +115,46 @@
     {DOMAIN: vol.Any(DEVICE_SCHEMA, PORT_SCHEMA)}, extra=vol.ALLOW_EXTRA
 )
 
+DATA_STORE = f"{DOMAIN}"
```

Comment:
```suggestion
DATA_STORE = DOMAIN
```

Reply:
Not even used.. how did that sneak passed me.